frontend/src/app/_services/collab.service.ts

```diff
@@ -87,4 +92,17 @@ export class CollabService implements OnInit, OnDestroy {
 
     return this.message$.asObservable();
   }
+
+  public requestChangeOfQuestion(question: Object) {
+    this.socket.emit("request", question);
+    this.request$.next([0, question]);
+  }
+
+  public getRequest = () => {
```

Comment:
Could I check that this gracefully handles the case that the other user does not agree

Reply:
Right now if the other user does not agree, a message is sent to the asking user that his request was rejected.